packages/flutter_tools/test/general.shard/coverage_collector_test.dart

```diff
@@ -2,8 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-@Timeout.factor(10)
```

Comment:
I added this in a previous PR and intended it to be temporary, but I forgot to remove it.